tools/buildtools/ambertools/parmconv.xml

```diff
@@ -0,0 +1,197 @@
+<tool id="parmconv" name="Convert Parameters" version="@VERSION@">
+  <description> to AMBER prmtop in preparation for MMPBSA</description>
+  <macros>
+    <import>macros.xml</import>
+  </macros>
+  <expand macro="requirements">
+    <requirement type="package" version="3.2.0">parmed</requirement>
+    <requirement type="package" version="2019.1">gromacs</requirement>
+    <requirement type="package" version="2.11.1">jinja2</requirement>
+  </expand>
+  <command detect_errors="exit_code">
+    <![CDATA[
+   bash '$bash_script' \$CONDA_PREFIX  &&
```

Comment:
you don't have CONDA_PRFIX in containers ....

can you use `which amber` ...

Reply:
yes testing now

Reply:
using which :+1: